tests/ui/privacy/private-variant-reexport.stderr

```diff
@@ -36,6 +36,11 @@ error: glob import doesn't reexport anything because no candidate is public enou
 LL |     pub use ::E::*;
    |             ^^^^^^
    |
+note: the most public imported item is `pub(crate)`
```

Comment:
Could you also update the main message and add a suggestion, as described in https://github.com/rust-lang/rust/pull/115993#issuecomment-1751723421?